x-pack/plugins/data_enhanced/public/search/async_search_strategy.test.ts

```diff
@@ -24,7 +24,11 @@ describe('Async search strategy', () => {
   beforeEach(() => {
     mockCoreSetup = coreMock.createSetup();
     mockDataStart = dataPluginMock.createStartContract();
-    (mockDataStart.search.getSearchStrategy as jest.Mock).mockReturnValue({ search: mockSearch });
+
+    const getStrategy = mockDataStart.search.getSearchStrategy as jest.Mock;
```

Comment:
I don't understand why we need to cast here. But since it's already there and not specific to this PR, I'm fine with it.